plugins/events-backend-module-bitbucket-server/README.md

```diff
@@ -0,0 +1,96 @@
+# events-backend-module-bitbucket-server
+
+Welcome to the `events-backend-module-bitbucket-server` backend plugin!
+
+This plugin is a module for the `events-backend` backend plugin
+and extends it with an `BitbucketServerEventRouter`.
+
+The event router will subscribe to the topic `bitbucketServer`
+and route the events to more concrete topics based on the value
+of the provided `x-event-key` metadata field.
+
+Examples:
+
+| x-event-key         | topic                               |
+| ------------------- | ----------------------------------- |
+| `repo:refs_changed` | `bitbucketServer.repo:refs_changed` |
+| `repo:modified`     | `bitbucketServer.repo:modified`     |
+| `pr:opened`         | `bitbucketServer.pr:opened`         |
```

Comment:
This seems like an odd event to include as a suggestion here. Do you have an example in mind where subscribing to `pr:open` would be beneficial to a backstage setup for entity refresh?

Reply:
To be honest no, I just had it as an example of an event for greater understanding of what the events were. If it's unnecessary I can remove

Reply:
@bforbis @davelil4 The event routers are independent of the "entity refresh" use case and only make events more specific (e.g., `bitbucketServer` -> `bitbucketServer.repo:refs_changed`). We have examples of other types of events at other event router docs as well.

What is then used by the entity provider depends on itself. Also, which events are received and then routed are based on your external (webhook) subscription.

Reply:
> @bforbis @davelil4 The event routers are independent of the "entity refresh" use case and only make events more specific (e.g., `bitbucketServer` -> `bitbucketServer.repo:refs_changed`). We have examples of other types of events at other event router docs as well.
> 
> What is then used by the entity provider depends on itself. Also, which events are received and then routed are based on your external (webhook) subscription.
> 
> PS: Not saying, we have to include it. We just don't need to remove it.

Yes, I assumed this was the case. It isn't necessary, just was included as an example of one of Bitbucket Server's webhook event keys. I removed the example in this last commit to fit what I thought you guys might prefer but I can undo this if it is actually preferred to be kept.

Reply:
I have no strong opinion one way or another if you think it's a useful hook to document (note, there are many others too in the shared doc link too which aren't documented either).